components/director/internal/tenantfetcher/converter_test.go

```diff
@@ -263,7 +265,7 @@ func TestConverter_EventToTenant(t *testing.T) {
 		out, err := conv.EventToTenant(eventsType, in)
 
 		// THEN
-		require.EqualError(t, err, "invalid event data format")
+		require.EqualError(t, err, apperrors.NewInvalidDataError("invalid event data format").Error())
```

Comment:
I don't know if there's any point in using these errors in fetcher which is not a part of Director API.